test/integration/dom.after.actions.test.ts

```diff
@@ -258,6 +258,78 @@ describe('Bpmn Elements registry - CSS class management', () => {
       bpmnVisualization.bpmnElementsRegistry.toggleCssClasses(nonExistingBpmnId, 'class1');
     });
   });
+
+  describe('Classes for Message Flow element and icon', () => {
+    // NOTE: temporary use a dedicated `BpmnVisualization` instance in each test
+    // Prevent bug to appear: classes are not purged from the internal CssRegistry on diagram load, so css classes added in a test are still present in the subsequent tests
```

Comment:
I have lately detected a bug while adding remove and toggle tests prior merging the PR. For reasons while we add the tests, see https://github.com/process-analytics/bpmn-visualization-js/pull/1624/files#r754459281
The bug is not related to message flow. I will open an issue soon about it.